src/main/scala/rise/eqsat/NamedRewrite.scala

```diff
@@ -414,7 +414,7 @@ object NamedRewriteDSL {
 
   import scala.language.implicitConversions
 
-  val `_`: rct.TypePlaceholder.type = rct.TypePlaceholder
+  val `__`: rct.TypePlaceholder.type = rct.TypePlaceholder
```

Comment:
maybe we can also just remove that syntax at that point and always use names ..

Reply:
That is completely up to you, I just renamed it in a way that Scala 3 does not compain about ...